pkg/observer/listener.go

```diff
@@ -7,14 +7,21 @@ import (
 	"encoding/gob"
 	"io"
 	"net"
+
+	"github.com/cilium/tetragon/api/v1/tetragon"
+	"github.com/cilium/tetragon/pkg/reader/notify"
 )
 
+type NotifyInterface interface {
```

Comment:
nit: maybe `MsgInterface` instead of `NotifyInterface`?

Reply:
hmm but its the Interface for the Notify()

Reply:
> hmm but its the Interface for the Notify()

I guess it's both the interface Notify() needs and what messages are required to implement (AFAIU). We can leave it as is. No strong feelings.